authorization-service/src/functions/basicAuthorizer/index.ts

```diff
@@ -0,0 +1,27 @@
+import schema from './schema';
+import { handlerPath } from '@libs/handler-resolver';
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [
+    {
+      http: {
+        method: 'get',
+        path: 'auth',
+        cors: true,
+        authorizer: {
```

Comment:
You need to make similar changes for existing import service lambda, not this place